core/jettyRunExtraFiles/mapfish-spring-application-context-override.xml

```diff
@@ -13,7 +13,7 @@
                                   location="classpath:mapfish-spring.properties"/>
 
     <bean id="mapPrinterFactory" class="org.mapfish.print.servlet.ServletMapPrinterFactory">
-        <property name="appsRootDirectory" value="${path_to_examples}"/>
+<!--        <property name="appsRootDirectory" value="${path_to_examples}"/>-->
```

Comment:
Wanted?

Reply:
It was no longer required. And as discussed the code throws an exception when the path points to non existent directory, which is what we want.